dotnet/year-service/Controllers/YearController.cs

```diff
@@ -8,7 +7,7 @@ namespace year_service.Controllers
     [ApiController]
     public class YearController : ControllerBase
     {
-        private static readonly int[] Years =

```

Comment:
Can this naming convention live in the editorconfig?

Reply:
[dotnet runtime editorconfig](https://github.com/dotnet/runtime/blob/32cbe969231ade2eb263befafdcf0380b7004645/.editorconfig#L75) has some rules that should come in handy for this...

```
dotnet_naming_rule.camel_case_for_private_internal_fields.style    = camel_case_underscore_style
dotnet_naming_style.camel_case_underscore_style.required_prefix = _
dotnet_naming_style.camel_case_underscore_style.capitalization = camel_case
```

Reply:
Copying the whole block from there. It's pretty standard.